pkg/blockchain/dbChainClient.go

```diff
@@ -0,0 +1,46 @@
+package blockchain
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type DatabaseChainClient struct{}
```

Comment:
Out of curiosity, why naming it DatabaseChainClient?

Reply:
The DatabaseChainClient will mock the blockchain by pulling payloads straight out of a database. For migration purposes, this database will be the one currently used by the old XMTP node.